docs/upgrading.md

```diff
@@ -1,66 +1,104 @@
-# Upgrading Contour to 0.15
+# Upgrading Contour
 
-## Changes in 0.15
-
-### Breaking change
-The major breaking change in Contour 0.15 is that `contour serve` (the mode that actually serves the control plane for Envoy) now requires that either TLS certificates be available, or you supply the `--insecure` parameter.
-
-**Without one of these options, Contour will not start.**
-
-### Other changes
+This document describes how to upgrade from Contour 0.14 to 0.15.
```

Comment:
This document describes the changes needed to upgrade your Contour installation.